wasm/src/ts/crypto/ed25519.ts

```diff
@@ -0,0 +1,25 @@
+import {getWASM, ensureReady} from './../wrapper';
```

Comment:
I think we can get by without writing TS wrapper code by hand. `wasm-pack` generates typedefs for interacting with the wasm module directly